uwrt_mars_rover/CMakeLists.txt

```
@@ -1,4 +1,60 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
```

```suggestion
cmake_minimum_required(VERSION 3.14.4)
```

Should change to match the min required version needed for required platforms as defined in REP 2000: https://www.ros.org/reps/rep-2000.html#foxy-fitzroy-may-2020-may-2023

---

done